packages/instant/src/components/buy_button.tsx

```diff
@@ -12,7 +15,8 @@ import { Button, Text } from './ui';
 export interface BuyButtonProps {
     buyQuote?: BuyQuote;
     assetBuyer?: AssetBuyer;
-    onAwaitingSignature: (buyQuote: BuyQuote) => void;
+    onPendingValidation: (buyQuote: BuyQuote) => void;
```

Comment:
can we reverse this to `onValidationPending` so that it is consistent with the other prop names?